fhir-server/liberty-config-tenants/config/tenant1/fhir-server-config.json

```
@@ -66,7 +151,13 @@
             },
             "membermatch": {
                 "enabled": false
-            }
+            },
+            "everything": [
+                "Location",
+                "Medication",
+                "Organization",
+                "Practitioner"
+            ]
```

I think I'd prefer to see this list of strings under 1 more level of nesting so that we have the flexibility of adding other everything config properties in the future:
```suggestion
            "everything": {
                "includeTypes": [
                    "Location",
                    "Medication",
                    "Organization",
                    "Practitioner"
                ]
            }
```

---

Done